src/clj/puppetlabs/services/master/file_serving.clj

```diff
@@ -130,9 +130,10 @@
   "Given the path of a project, the name of a module, and the modulepath defined
   in the project's config (can be nil) search the project's module path and
   return the path to that module as a File, or nil."
-  [project-root module modulepath]
-  (->> project-root
-       (dirs-in-project-modulepath modulepath)
+  [bolt-builtin-content-dir project-root module modulepath]
+  (->> bolt-builtin-content-dir
```

Comment:
It's unlikely the builtin content will change, so we could move this into a variable. However since this computation's pretty cheap and since it is possible to dynamically change the builtin content (doesn't mean users should), I lean towards keeping it as-is.